test/src/test/java/hudson/cli/DisablePluginCommandTest.java

```diff
@@ -138,6 +182,159 @@ public void disablePluginsWithError16() {
         assertJenkinsInQuietMode(); // some plugins were disabled
     }
 
+    /**
+     * All the dependant plugins, mandatory or optional, are disabled using <i>-strategy all</i>.
+     */
+    @Test
+    @Issue("JENKINS-27177")
+    @WithPlugin({"variant.hpi", "depender-0.0.2.hpi", "mandatory-depender-0.0.2.hpi", "plugin-first.hpi", "dependee-0.0.2.hpi", })
+    public void disablePluginsStrategyAll() {
```

Comment:
`StrategyAll` please use the same name for the previous tests, the `St` is not really good as abbreviation from my PoV.

Reply:
I used it running away from large method names. What a paradox, because I really like it